app-frontend/src/assets/styles/sass/components/_form.scss

```diff
@@ -209,6 +209,10 @@ input[type=radio] {
     background: none;
     color: $shade-dark;
     box-shadow: none;
+
+    &.darkBg {
```

Comment:
css class names are usually done in lower case with dashes like `dark-background`. Also better to be verbose than concise for things like this